### DISCUSSION TITLE
🤖 Cochat

### DISCUSSION THREAD

**Discussion by mcclowes** [codatio/codat-docs#398] [Show and tell]
![Screenshot 2023-05-22 at 12 53 40](https://github.com/codatio/codat-docs/assets/4573407/11790c6d-42e0-41f7-9772-5cee87308cac)

Having trouble getting started with Codat? Get instant assistance with [Cochat](https://cochat-test.azurewebsites.net/) our beta AI tool trained on the Codat docs.

Cochat is an advanced natural language processing (NLP) tool developed to enhance the user experience of Codat customers. By leveraging the power of GPT 3.5, Cochat provides an interactive and conversational interface to help users make sense of the Codat documentation and better understand how to leverage its capabilities for building integrations.

 Try [asking it a question](https://cochat-test.azurewebsites.net/) or [read more](https://docs.codat.io/other/cochat).

**Comment by jasond-s** [comment]
The link for cochat appears to be broken.
